### PR TITLE
ccfri 4004, and other small ui bug fixes

### PR DESCRIPTION
**Pull request by jenbeckett**
Fix so that some Org Information does not get cut off on the summary Dec page. I also added a max char length for the affected fields so it will not run over. Quick google says these fields are generally 40 chars so I made it 60 to be safe. 

Other fixes:
https://eccbc.atlassian.net/browse/CCFRI-4270
Update checkboxes on Family / Group CCOF pages to use the values 1 and 0 and not default to Boolean values. This will let the checkboxes populate correctly after saving then refreshing.

Landing page had an error with an old LoDash import. This error was preventing some facilities with MTFI from loading properly.

Action Required text on Report Changes page used to be red. Due to the vue3 update to the data-table, it was not rendering properly. 

I made a bug ticket for this: https://eccbc.atlassian.net/browse/CCFRI-4280

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/store/ccof/facility.js [hunk 1]
```diff
@@ -117,133 +103,124 @@ export const useFacilityStore = defineStore('facility', {
...
-            throw error;
-          }
+          return response;
+        } catch (error) {
+          console.log(`Failed to save new Facility - ${error}`);
+          throw error;
+        }
+      } else {
+        try {
+          let response = await ApiService.apiAxios.post(ApiRoutes.FACILITY, payload);
```
Comment (weskubo-cgi) [inline frontend/src/store/ccof/facility.js:199]: This should be const.

File: frontend/src/components/ccofApplication/group/OrganizationInformation.vue [hunk 2]
```diff
@@ -61,7 +61,7 @@
                     :disabled="isLocked"
                     variant="outlined"
                     required
-                    :rules="rules.required"
+                    :rules="[...rules.required, rules.maxLength(MAX_CHAR_LENGTH)]"
```
Comment (weskubo-cgi) [inline frontend/src/components/ccofApplication/group/OrganizationInformation.vue:64]: As discussed, let's remove this validation. I'm just cautious of implications for existing data.

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/components/ccofApplication/group/OrganizationInformation.vue (hunk 2 above)
Comment (jenbeckett) [reply frontend/src/components/ccofApplication/group/OrganizationInformation.vue:64]: they gone

File: frontend/src/store/ccof/facility.js (hunk 1 above)
Comment (jenbeckett) [reply frontend/src/store/ccof/facility.js:199]: I always miss one!! it's fixed now :)

File: frontend/src/store/navBar.js [hunk 3]
```diff
@@ -196,7 +195,7 @@ export const useNavBarStore = defineStore('navBar', {
      * and reforce the navbar to refresh
      ************************************************/
     setNavBarValue({ facilityId, property, value }) {
-      console.log('set navBar called with value: ' + property + ' , ' + value);
+      //console.log('set navBar called with value: ' + property + ' , ' + value);
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/store/navBar.js:198]: Sure - it's gone

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**